### DISCUSSION TITLE
在组件之外怎么使用react-router-dom的导航

### DISCUSSION THREAD

**Discussion by caihai123** [remix-run/remix#5897] [Q&A]
我现在正在封装axios拦截器，我需要在发现403的时候跳转到登录页，我应该怎么做？我使用的react-router-dom@6.8.0

**Comment by lili21** [comment]
You can do it inside loader with `redirect`.

**Reply by caihai123** [reply]
从文档上看好像确实可以，但是我刚才试了一下，执行之后没反应，我不知道为什么
